episodes/20-reflecting.md

```diff
@@ -46,24 +46,41 @@ the trial run are used to improve the quality of your lesson.
 
 ## Discussion: trial runs (35 minutes)
 
-Look back at the notes you took after you completed your trial run (5 mins)
+Look back at the notes you took and feedback you received during your trial run (5 mins).
 
 Now report out to the other participants about that trial run.
 Try to answer the following questions:
```

Comment:
```suggestion
Try to answer some or all of the following questions:
```

Reply:
I like all the extra questions you added, but now the list is longer I think it is reasonable not to expect every participant to address every question in their report out.